cache4k/src/commonMain/kotlin/io/github/reactivecircus/cache4k/RealCache.kt

```diff
@@ -234,14 +228,14 @@ internal class RealCache<Key : Any, Value : Any>(
     private fun recordWrite(cacheEntry: CacheEntry<Key, Value>) {
         if (expiresAfterAccess) {
             val accessTimeMark = cacheEntry.accessTimeMark.value
-            cacheEntry.accessTimeMark.set(accessTimeMark + accessTimeMark.elapsedNow())
+            cacheEntry.accessTimeMark.value = (accessTimeMark + accessTimeMark.elapsedNow())
         }
         if (expiresAfterWrite) {
             val writeTimeMark = cacheEntry.writeTimeMark.value
-            cacheEntry.writeTimeMark.set(writeTimeMark + writeTimeMark.elapsedNow())
+            cacheEntry.writeTimeMark.value = (writeTimeMark + writeTimeMark.elapsedNow())
```

Comment:
these aren't complex expression but perhaps we can also use [update(function: (T) -> T)](https://github.com/Kotlin/kotlinx.atomicfu/blob/450b056efa4c8e4f922f2a3f8d1958404513b1f7/atomicfu/src/commonMain/kotlin/kotlinx/atomicfu/AtomicFU.common.kt#L163) just in case.